<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EasyRemoteDesktopClass</class>
 <widget class="QWidget" name="EasyRemoteDesktopClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EasyRemoteDesktop</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>560</width>
     <height>380</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabBar::tab{width:280}
QTabBar::tab{height:30}
QTabBar{font: bold;}
QTabBar{font-size: 14px;}

QTabWidget::pane{border-width: 1px; border-style: solid; border-color: rgb(230, 230, 230); border-top: 1px solid #bbbbbb;}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="serviceSettingTab">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QSplitter" name="splitter">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>280</y>
       <width>450</width>
       <height>35</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="serverPreviewButton">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="serverSentButton">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLineEdit" name="serverPortEdit">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>9001</string>
        </property>
        <property name="maxLength">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="serverPortLabel">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>端号：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="serverProtocolLabe">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>协议：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="serverProtocolComboBox">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>TCP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>UDP</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="serverFpsLabel">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>帧率</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="serverFpsComboBox">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>15</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>25</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="ShowPreviewWindow" name="serverPreviewWindow">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>400</width>
       <height>250</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ClientSettingTab">
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ShowPreviewWindow</class>
   <extends>QOpenGLWidget</extends>
   <header>ShowPreviewWindow.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="EasyRemoteDesktop.qrc"/>
 </resources>
 <connections/>
</ui>
